metrics-scraper/config.go

```diff
@@ -0,0 +1,37 @@
+package metricsscraper
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type Config struct {
+	PollRate Duration     `json:"poll_rate"`
+	Nodes    []NodeConfig `json:"nodes"`
+	ChainID  string       `json:"chain_id"`
+}
+
+type NodeConfig struct {
+	RPCURL   string `json:"rpc_url"`
+	CometURL string `json:"comet_url"`
```

Comment:
Seems this is unused (at least for now). Can we remove it?

Reply:
addressed in: 1395e6d260fa907ffca176182da7f1880f65fa12